content/chainguard/chainguard-images/features/private-apk-repos/index.md

```diff
@@ -107,6 +107,36 @@ apk update
 
 Following that, you can proceed to search and install packages from your private APK repository.
 
+## Generating a Pull Token for Authentication 
+
+To generate a longer lived token for authentication purposes, it is necessary to create a pull token to access your private APK repository. 
+
+In this example we set up a pull token that expires in 35 days, to use with CI. 
+
+**Generate a pull token:**
+
+```shell
+chainctl auth pull-token --parent=<YOUR_CHAINGUARD_ORG> --ttl=850h -o json > /tmp/token.json
```

Comment:
Not sure about this but I think this should support the env output option and then this would be better - see https://edu.chainguard.dev/chainguard/libraries/access/#use-environment-variables  .. can you please check and if that works update the docs

Reply:
Both here and later replace <YOUR_CHAINGUARD_ORG> with ORGANIZATION for consistency with the rest of the document.